package/crossplane.yaml

```diff
@@ -2,3 +2,5 @@ apiVersion: meta.pkg.crossplane.io/v1alpha1
 kind: Provider
 metadata:
   name: provider-aviatrix
+  annotations: 
+   meta.crossplane.io/description: Upbound Partner Provider for configuring Aviatrix Secure Multi Cloud functionality
```

Comment:
@therealmitchconnors can we add all of the supported object annotations here? 

Note that the readme annotation is what will get rendered in the provider's detail page. The description annotation is what appears in search results.

Feel free to use https://github.com/crossplane-contrib/provider-aws/blob/master/package/crossplane.yaml as a reference.